api/tests/certification/results/unit/domain/models/PrivateCertificate_test.js

```diff
@@ -1,7 +1,10 @@
 import { PrivateCertificate } from '../../../../../../src/certification/results/domain/models/PrivateCertificate.js';
 import { AutoJuryCommentKeys } from '../../../../../../src/certification/shared/domain/models/JuryComment.js';
 import { SESSIONS_VERSIONS } from '../../../../../../src/certification/shared/domain/models/SessionVersion.js';
-import { status as assessmentResultStatuses } from '../../../../../../src/shared/domain/models/AssessmentResult.js';
+import {
+  AssessmentResult,
+  status as assessmentResultStatuses,
+} from '../../../../../../src/shared/domain/models/AssessmentResult.js';
```

Comment:
```suggestion
import {
  AssessmentResult,
  status as assessmentResultStatuses,
} from '../../../../../../src/shared/domain/models/AssessmentResult.js';
```

**question:** Au final on importe `AssessmentResult` pour les status qu'on importe déjà en dessous.
Choisir l'un ou l'autre ?

Reply:
done !